client/stylesheets/_toCat.scss

```diff
@@ -5445,4 +5445,33 @@ On click the images will be displayed at normal size to complete the effect
 
 .listingPayCash i {
 	margin-right: 5px;
-}
\ No newline at end of file
+}
+
+
+/** Amount Profile **/
+
+.profileUserAmount ul {
+    margin: 8px 0 !important;
+}
+
+.profileUserAmount ul li {
+    display: inline-block;
+    font-size: 0.9rem;
+    padding: 10px 15px;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces